content/2017/03/21/blocking-analytics-spam.rst

```diff
@@ -0,0 +1,145 @@
+public: yes
+author: jonny
+tags: [
+  Spam,
+  DevOps,
+  'Build Tools',
+  'Front-end',
+  JavaScript,
+  Analytics,
+  Code
+  ]
+image:
+  - src: '2017/server-client/spa2.jpg'
+summary: |
+  Google Analytics is great for gathering data on who uses your web
+  application, but becomes worthless if spam sessions start infesting your
+  data. Here's how we've tried to combat the problem for `oddbird.net`_.
+
+  .. _oddbird.net: /
```

Comment:
I'm not sure I even understand the question, but I'm fine using a full link here.